dependencies.gradle

```diff
@@ -10,7 +10,7 @@ dependencies {
     compileOnly('com.github.GTNewHorizons:Jabba:1.2.21:dev') {transitive = false}
     compileOnly('com.github.GTNewHorizons:inventory-tweaks:1.5.15:api') {transitive = false}
     compileOnly('com.github.GTNewHorizons:OpenComputers:1.7.5.32-GTNH:api') {transitive = false}
-    compileOnly('com.github.GTNewHorizons:waila:1.5.21:dev') {transitive = false}
+    compile('com.github.GTNewHorizons:waila:1.5.21:dev') {transitive = false}
```

Comment:
Did you leave this in by accident? Waila should be an optional dependency here (compileOnly)

Reply:
sorry waila dev is optional, but waila api is not
fixed

Reply:
![image](https://user-images.githubusercontent.com/33802162/218227739-32b6fc58-81e0-4ce2-9928-914c4e3197a1.png)